### ISSUE TITLE
is support vs2017?

### ISSUE THREAD

**Issue by CaiHuan** [chromium/vs-chromium#28]
is support vs2017?

**Comment by randomascii** [comment]
The installer warns that VS 2017 is not supported and that installing VsChromium for VS 2017 could cause instability. I'm trying to move towards using VS 2017 and it would be _great_ to get VS Chromium support for VS 2017.

**Comment by rpaquay** [comment]
I did some testing, and VsChromium seems to work fine with VS 2017. The warning from the installer is just that... a warning. The reason there is a warning is that the vsix manifest format has been updated for VS 2017, and VS 2017 "blindly" warns when trying to use any extension that does not expose the new format (source: http://www.visualstudioextensibility.com/2017/01/10/its-time-to-change-the-vsix-manifest-of-your-extension-to-v3-for-visual-studio-2017-compatibility/).

I need to spend some time to figure out how to upgrade the manifest format, but VS 2017 hung when I tried to load the vs-chromium solution (after a failed conversion), so it does not look like it is as easy/trivial as I had hoped.
